pynicotine/cli.py

```diff
@@ -44,9 +45,13 @@ def __init__(self):
         self.has_custom_prompt = False
         self.prompt_message = ""
         self.prompt_callback = None
+        self.prompt_silent = False
 
     def run(self):
 
+        # Small time window for setup prompts on startup
+        time.sleep(1)
```

Comment:
Is it enough to use the existing `sleep` on Line 64 by moving it to the start of the `while` loop.